decentraland/unity-explorer
# fix: introduce developer artificial delay for testing purposes

## What does this PR change?

Introduces a feature to enable artificial delay for web requests. It helps to debug issues caused by the slow internet connection

## How to test the changes?

Take a look for plugin settings and tweak options to see the difference in loading speed
<img width="335" alt="image" src="https://github.com/decentraland/unity-explorer/assets/71646502/5223d532-34c3-4705-91e5-456c489261a6">

1. Enable/Disable delay
2. Launch the explorer

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
